manifests/init.pp

```diff
@@ -0,0 +1,10 @@
+class robottelo_slave () {
+
+  include ::slave
+  include ::libvirt
```

Comment:
I wasn't sure what to do about that given they live in foreman-inra/puppet/modules and don't have associated git repositories for them individually or their own metadata.